usecases/auth/authorization/types.go

```diff
@@ -85,13 +85,18 @@ func Roles(roles ...string) []string {
 //
 //	A slice of strings representing the resource paths.
 func Collections(classes ...string) []string {
-	if len(classes) == 0 || (len(classes) == 1 && classes[0] == "") {
+	if len(classes) == 0 || (len(classes) == 1 && (classes[0] == "" || classes[0] == "*")) {
 		return []string{"collections/*"}
 	}
 
 	resources := make([]string, len(classes))
 	for idx := range classes {
-		resources[idx] = fmt.Sprintf("collections/%s", classes[idx])
+		if classes[idx] == "" {
+			resources[idx] = "collections/*"
+		} else {
+			resources[idx] = fmt.Sprintf("collections/%s/*", classes[idx])
+		}
+		resources[idx] = fmt.Sprintf("collections/%s/*", classes[idx])
```

Comment:
**Issue (blocking):** will this will overwrite the assignment on L95 to be `collections//*`?